kong/ci/kong.rake

```diff
@@ -0,0 +1,92 @@
+require 'ci/common'
+
+def kong_version
+  ENV['FLAVOR_VERSION'] || '0.8.1'
+end
+
+def kong_rootdir
+  "#{ENV['INTEGRATIONS_DIR']}/kong_#{kong_version}"
+end
+
+container_name='dd-test-kong'
+container_name_db="#{container_name}-database"
+container_port1=8000
+container_port2=8443
+container_port3=8001
+container_port4=7946
+
+def wait_on_logs(c_name, *include_array)
```

Comment:
Going to add this to the `gem`, it would be great if you could refactor to that once it's merged 😊 (it would be a nice example for contributors).